readme.md

```diff
@@ -117,6 +117,31 @@ Define a function `makeCar` that takes two parameters - `model` and `color` - an
 var celica = makeCar("Toy-Yoda Celica", "limegreen");
 ```
 
+<details>
+  <summary><strong>Solution...</strong></summary>
+
+  ```js
+  // ES5
+  function makeCar(model, color){
+    return {
+      model: model,
+      color: color
+    }
+  }
+  ```
+
+  ```js
+  // ES6
+  function makeCar(model, color){
+    return {
+      model: model,
+      color: color
+    }
+  }
+  ```
+
+</details>
+
 This is the basic idea behind OOP: we define a blueprint for an object and use it to generate multiple instances of it!
 
```

Comment:
maybe note that instance is a technical term and it IS an object. whereas a class is NOT an object, but merely a blueprint.